pio/src/iSpindel.cpp

```diff
@@ -393,8 +392,11 @@ bool startConfiguration()
   wifiManager.setConfSSID(htmlencode(my_ssid));
   wifiManager.setConfPSK(htmlencode(my_psk));
 
+  char my_ap_ssid[20];
+  sprintf(my_ap_ssid, "iSpindel_%06X",ESP.getChipId());
+
   CONSOLELN(F("started Portal"));
-  wifiManager.startConfigPortal(autoconf_ssid.c_str());
+  wifiManager.startConfigPortal(my_ap_ssid);
 
```

Comment:
wifiManager.startConfigPortal("iSpindel_"+String(ESP.getChipId()));

Reply:
This will return a StringSumHelper and not a char*

Reply:
made a couple of tests, and while the diff is not a lot the inline String is the cheapest one:
```suggestion
  wifiManager.startConfigPortal((String("iSpindel_") + ESP.getChipId()).c_str());
```

> String literal
> RAM:   [=====     ]  47.7% (used 39092 bytes from 81920 bytes)
> Flash: [====      ]  44.1% (used 461032 bytes from 1044464 bytes)
> 
> String var
> RAM:   [=====     ]  47.7% (used 39092 bytes from 81920 bytes)
> Flash: [====      ]  44.1% (used 461064 bytes from 1044464 bytes)
> 
> char[]
> RAM:   [=====     ]  47.7% (used 39104 bytes from 81920 bytes)
> Flash: [====      ]  44.1% (used 461052 bytes from 1044464 bytes)
>

Reply:
I updated according to your suggestion.
This will decrease ram usage, however it will just dynamically allocate the value and put it on the heap and increase fragmentation. As it is a value with a known fixed length I personally think a char array fits better.

Reply:
Not opposed to canonical char[] at all, but then lets reduce it to the minimum which is also the leanest of all

> RAM:   [=====     ]  47.7% (used 39104 bytes from 81920 bytes)
Flash: [====      ]  44.1% (used 460972 bytes from 1044464 bytes)
```suggestion
  char ssid[14];
  snprintf(ssid, sizeof(ssid), "iSpindel_%X", ESP.getChipId());
  wifiManager.startConfigPortal(ssid);
```